homeassistant/components/canary/camera.py

```diff
@@ -127,11 +127,14 @@ def motion_detection_enabled(self):
     async def async_camera_image(self):
         """Return a still image response from the camera."""
         await self.hass.async_add_executor_job(self.renew_live_stream_session)
+        live_stream_url = await self.hass.async_add_executor_job(
+            self._live_stream_session.live_stream_url
```

Comment:
This is a property right? In that case you need to use getattr

Reply:
Right. Adjusted.